client/extensions/woocommerce/index.js

```diff
@@ -10,7 +10,7 @@ import config from 'config';
  */
 import { navigation, siteSelection } from 'my-sites/controller';
 import { renderWithReduxStore } from 'lib/react-helpers';
-import ProductForm from './app/products/product-form';
+import ProductCreate from './app/products/product-create';
```

Comment:
Basically my first comment on the PR - do we really want to have separate containers for creation and editing, or could we continue using ProductForm and have it understand new and existing products instead?